packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -427,22 +478,43 @@ class _TableRowState<T> extends State<_TableRow<T>>
   @override
   Widget build(BuildContext context) {
     final row = tableRowFor(context);
-    return AnimatedBuilder(
-      animation: showController,
-      builder: (context, child) {
-        return SizedBox(
-          height: _Table.defaultRowHeight * showAnimation.value,
-          child: Material(
-            child: child,
-            color: widget.backgroundColor ?? Theme.of(context).canvasColor,
-          ),
-        );
-      },
-      child: InkWell(
-        onTap: () => widget.onPressed(widget.node),
-        child: row,
+
+    final box = SizedBox(
+      height: _Table.defaultRowHeight,
+      child: Material(
+        color: widget.backgroundColor ?? Theme.of(context).canvasColor,
+        child: InkWell(
+          key: contentKey,
+          onTap: () => widget.onPressed(widget.node),
+          child: row,
+        ),
       ),
     );
+    if (widget.expansionChildren == null)
+      return box;
```

Comment:
nit: put return statement on same line.

Reply:
This code is dart-formatted.   Removing the else block has made this change.